.vscode/settings.json

```diff
@@ -1,39 +1,39 @@
 {
     "editor.formatOnSave": true,
-    "files.exclude": {
-        ".pytest_cache/**/*": true,
-        "**/*.pyc": true,
-        "**/locust.wpr": true,
-        "locust.egg-info/**": true,
-        "locustio.egg-info/**": true,
-        "docs/_build/**": true,
-        "docs/cli-help-output.txt": true,
-        "docs/config-options.rst": true,
-        "**/mock.*.egg": true,
-        "dist/**": true,
-        ".idea/**": true,
-        "**/*.iml": true,
-        "**/*.ipr": true,
-        "**/.vagrant": true,
-        "**/build/": true,
-        "**/.coverage": true,
-        "**/.tox/": true,
-        "docs/env-options.rst": true,
-        "**/.editorconfig": true,
-        "**/__pycache__": true,
-        "locust/_version.py": true,
-        "**/web_test_*.csv": true,
-        "**/.eggs/": true,
-        "**/.pytest_cache": true,
-        "**/.sass-cache/": true,
-        "locust/contrib/.mypy_cache/**/*": true,
-        "locust/user/.mypy_cache/**/*": true,
-        ".mypy_cache/**/*": true,
-        "locust/test/mock_*.py": true,
-        "**/err.txt": true,
-        "**/out.txt": true,
-        "**/.env": true
-    },
+    // "files.exclude": {
```

Comment:
Was this an accidental change? Some of these might no longer be relevant, but I'm pretty sure they don't ALL deserve to be removed :)

Reply:
Hey sorry yeah I'll revert this, I don't work like this locally as I like to see all the hidden files (versioning one, etc.) but clearly did a careless commit all here 😱